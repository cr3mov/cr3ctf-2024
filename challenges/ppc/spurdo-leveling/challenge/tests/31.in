247334499407654463 913328202340557023
438936 779591
438936 3071435
