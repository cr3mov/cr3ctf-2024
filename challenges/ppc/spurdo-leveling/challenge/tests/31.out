16757300503524806 168587904219 168587904219
